lib/main_containers.pm

```diff
@@ -54,7 +54,14 @@ sub load_image_test {
     loadtest('containers/image', run_args => $run_args, name => 'image_' . $run_args->{runtime});
 }
 
-sub load_3rd_party_image_test {
+sub load_third_party_image_test {
+    # Third party image tests are not needed or redundant on:
+    # - Staging
+    # - Public Cloud
+    # - Testing latest Tumbleweed image with CONTAINER_IMAGE_TO_TEST
+    # - FIPS tests
+    # - Testing alternative cgroups version
+    return unless (is_staging || is_public_cloud || get_var("CONTAINER_IMAGE_TO_TEST") || get_var("FIPS_ENABLED") || get_var("CONTAINERS_CGROUP_VERSION"));
```

Comment:
Please add `is_vmware`, the containers fails because lack of cpu version support.
https://openqa.suse.de/tests/18850015#step/podman_3rd_party_images/340